src/selectors/events.js

```diff
@@ -58,55 +55,50 @@ export const groupEventsByStartTime = (events: Event[]): EventDays => {
     : sections.days;
 };
 
-const generateRecurringEvent = event => recurrance => {
-  const [recurranceDay, recurrancyMonth, recurranceYear] = recurrance.split(
+const generateRecurringEvent = event => recurrence => {
+  const [recurrenceDay, recurrenceMonth, recurrenceYear] = recurrence.split(
     "/"
   );
-  const [eventStartDate, eventStartTime] = event.fields.startTime[locale].split(
-    "T"
-  );
-  const eventEndTime = event.fields.endTime[locale].split("T")[1];
-  const [eventStartYear, eventStartMonth, eventStartDay] = eventStartDate.split(
-    "-"
+
+  const startTime = DateTime.fromFormat(
+    event.fields.startTime[locale],
+    contentfulFormat,
+    { setZone: true }
```

Comment:
How do these options compare to using `DateTime.fromISO(<contentfulString>)`? On my `luxon` branch this seems to work, but I probably missed some detail

Reply:
Hmm ok. I'll bear this all in mind for my `luxon` branch. Also I found we can make our tests more predictable by setting the env var `TZ=UTC` on a test run. Which makes changing the tests a more viable option. I'll do that on my branch too.